node/api/client/get.go

```diff
@@ -0,0 +1,23 @@
+package client
+
+import (
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/node/api"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// GetConsensus requests the /consensus api resource
+func (c *Client) GetConsensus() (cg api.ConsensusGET, err error) {
```

Comment:
Need to swap the verb here. `ConsensusGet`. Same for everything else. We do this so when it gets sorted alphabetically, you don't have to search through a bunch of `Get` prefixes to find what you want.